AppStore/AppStore/IntroductionViewController.swift

```diff
@@ -0,0 +1,103 @@
+        case .top:
+            if let subtitle = subtitle {
+                subtitleLabel.text = subtitle
+            }
+            subtitleLabel.applyFontStyle(.subtitle, isDark: isDark)
+            titleLabel.applyFontStyle(.title, isDark: isDark)
+
+            NSLayoutConstraint.activate([
+                subtitleLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 10),
+                subtitleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: viewPadding),
+                subtitleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -viewPadding),
+                subtitleLabel.heightAnchor.constraint(equalToConstant: 20),
+                titleLabel.topAnchor.constraint(equalTo: subtitleLabel.bottomAnchor, constant: 5),
+                titleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: viewPadding),
+                titleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -40),
+                titleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: 10)
+            ])
+
+        case .middle:
+            titleLabel.applyFontStyle(.largeTitle, isDark: isDark)
+
+            NSLayoutConstraint.activate([
+                titleLabel.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -100),
+                titleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: viewPadding),
+                titleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -190),
+                titleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: 10)
+            ])
+
+        case .bottom:
+            if let subtitle = subtitle {
+                subtitleLabel.text = subtitle
+            }
+            subtitleLabel.applyFontStyle(.subtitle, isDark: isDark)
+            titleLabel.applyFontStyle(.title, isDark: isDark)
+
+            NSLayoutConstraint.activate([
+                titleLabel.bottomAnchor.constraint(equalTo: descriptionLabel.topAnchor, constant: -5),
+                titleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: viewPadding),
+                titleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: viewPadding),
+                titleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: 10),
+                subtitleLabel.bottomAnchor.constraint(equalTo: titleLabel.topAnchor, constant: -3),
+                subtitleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: viewPadding),
+                subtitleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: viewPadding),
+                subtitleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: 10)
+            ])
+        }
+    }
+
+    private func setDescription(description: String) {
+        descriptionLabel.text = description
+        descriptionLabel.applyFontStyle(.description, isDark: true)
```

Comment:
```suggestion
    private func setDescription(description: String, isDark: Bool) {
        descriptionLabel.text = description
        descriptionLabel.applyFontStyle(.description, isDark: isDark)
```
description 설정하는 부분만 dark 모드 제어가 안 되있는 것 같아 혹시 위와 같이 바꿔야 할까 여쭈어 봅니다!

Reply:
헉...! 꼼꼼하게 봐주셔서 감사합니다! 
완전 놓치고 있었네요